src/main/java/Deadline.java

```diff
@@ -0,0 +1,12 @@
+public class Deadline extends Task{
```

Comment:
Can add a space between Task and the curly bracket { to fit with the coding standard